OTAnalytics/plugin_ui/cli.py

```diff
@@ -342,6 +385,25 @@ def _get_sections_file(file: str) -> Path:
 
         return sections_file
 
+    def _export_events(self, sections: Iterable[Section], save_path: Path) -> None:
+        events = self._event_repository.get_all()
+        match self.cli_args.event_format:
+            case EventFormat.CSV:
+                exporter = self._event_list_export_formats[OTC_CSV_FORMAT_NAME]
+                actual_save_path = save_path.with_suffix(f".{EventFormat.CSV.value}")
+                exporter.export(events, sections, actual_save_path)
+            case EventFormat.EXCEL:
+                exporter = self._event_list_export_formats[OTC_EXCEL_FORMAT_NAME]
+                actual_save_path = save_path.with_suffix(f".{EventFormat.EXCEL.value}")
+                exporter.export(events, sections, actual_save_path)
+            case _:
+                exporter = self._event_list_export_formats[OTC_OTEVENTS_FORMAT_NAME]
+                actual_save_path = save_path.with_suffix(
+                    f".{EventFormat.OTEVENTS.value}"
+                )
```

Comment:
This is three times nearly the same code. If we save an instance of `EventlistExporter` in CliArguments, we can directly use the instance here and get rid of the `match ... case ...`.